ada_feeding_perception/ada_feeding_perception/face_detection.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python3
+
+"""
+This file defines the FaceDetection class, which publishes a 3d PointStamped location
+of a detected mouth with respect to camera_depth_optical_frame.
+"""
+from ada_feeding_msgs.msg import FaceDetection
+from ada_feeding_msgs.srv import ToggleFaceDetection
+import cv2
+from cv_bridge import CvBridge
+from geometry_msgs.msg import PointStamped
+from geometry_msgs.msg import Point
+import rclpy
+from rclpy.node import Node
+from sensor_msgs.msg import Image
+import os
+from threading import Lock
+from typing import Tuple
+from rclpy.parameter import Parameter
+from rcl_interfaces.msg import ParameterDescriptor, ParameterType
+
+
+from ada_feeding_perception.helpers import (
+    download_checkpoint,
+)
+
+
+class FaceDetectionNode(Node):
```

Comment:
Technically, for best coding practices you should have a class docstring as well. Can you add one below the class definition? Feel free to have it be mostly the same as your module docstring at the top of the file.